code/game/jobs/job/1_captain.dm

```diff
@@ -0,0 +1,328 @@
+var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
+
+/datum/job/captain
```

Comment:
why's this file called 1_captain when it contains jobs that aren't captain? rename it to 1_command

Reply:
I thought of that, but other servers have the same naming convention for the same file.

Don't really want to make it harder for muscle memory.